src/versions/v1/SandySounds.js

```diff
@@ -36,12 +36,14 @@ class SandySounds extends EventEmitter {
         let node = new Node({
             host: options.host,
             port: options.port,
+            restPort: options.restPort,
             region: options.region,
             numShards: options.numShards,
             userId: options.userId,
             password: options.password,
         });
 
+        node.connect();
```

Comment:
Not needed as node's call their own connect method in constructor

Reply:
👍 I'll make that change